alice-ide/src/main/resources/org/alice/stageide/themes/FlatLaf.properties

```diff
@@ -0,0 +1,67 @@
+#todo, turn off maven checkstyle on this file.
+###################################################################
+#General UI color choices/overrides
+###################################################################
+#@foreground =
+#@background =
+
+# this controls the ui highlight in dropdowns n'at
+[light]@accentColor = rgb(16, 113, 158)
+[dark]@accentColor= rgb(26, 113, 103)
+
+@accentFocusColor = saturate(@accentColor, 25%)
+@buttonArrowColor = @accentColor
+@buttonHoverArrowColor = @accentFocusColor
+Component.arrowType = triangle
+
+###################################################################
+# Alice specific variables
+###################################################################
+Alice.Comment.Color = #ddd
+Alice.Comment.Color.foreground = #060
+
+# that yellow color for objects/classes
+[light]Alice.Noun.Color = #FDF6C0
+[dark]Alice.Noun.Color =  rgb(211, 174, 57)
+
+# types, currently the same as nouns. What is a type? what should this be?
+[light]Alice.Type.Color = $Alice.Noun.Color
+[dark]Alice.Type.Color =  $Alice.Noun.Color
+Alice.Type.Color.muted = desaturate($Alice.Type.Color, 10%)
+
+# logically, Constructor color should relate to nouns, so making that more explicit
+[light]Alice.Constructor.Color = spin($Alice.Noun.Color, -10)
```

Comment:
`spin` makes me think of a random value. Mayb add a comment to clarify what it does?

Reply:
this is going away with the new colors